crates/matrix-sdk/src/send_queue.rs

```diff
@@ -863,14 +863,21 @@ impl SendHandle {
     ///
     /// Returns true if the sending could be aborted, false if not (i.e. the
     /// event had already been sent).
+    #[instrument]
     pub async fn abort(self) -> Result<bool, RoomSendQueueStorageError> {
+        trace!(tnx_id = %self.transaction_id, "received an abort request");
```

Comment:
That's already part of the span because `instrument` will attach whatever is in `self` to the span by default.

Reply:
Thanks! Will probably skip `self`, and manually record the fields I'm interested in at the `instrument` derive level.